services/apps/alcs/src/providers/typeorm/migrations/1693945647800-rename_nfu_placement.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class renameNfuPlacement1693945647800 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+          `ALTER TABLE "alcs"."application_submission" RENAME COLUMN "nfu_total_fill_placement" TO "nfu_total_fill_area"`,
```

Comment:
This will break API. You need to update all corresponding *.ts files that reference this column.

Reply:
![image](https://github.com/bcgov/alcs/assets/26191672/204e45b6-3bf8-4ea5-97d8-4103d41e0c53)
Appears that the only reference is in this file